frontend/src/Pages/TeamDebateRoom.tsx

```
@@ -0,0 +1,1551 @@
+            setPlayerReadyStatus(prev => {
+              const updated = new Map(prev);
+              Object.entries(team2Status).forEach(([userId, ready]) => {
+                updated.set(userId, ready);
+              });
+              return updated;
+            });
+          }
+          
+          // Check if opponent team members are all ready (but don't override localReady)
+          // localReady should only be set when the user clicks the ready button
+          const opponentReadyRaw = isTeam1Latest ? data.team2Ready : data.team1Ready;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)